examples/data-solutions/dp-foundation/demo/gcs2bq.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# --------------------------------------------------------------------------------
```

Comment:
Where is the original source for this? We should link to the source, otherwise we need to keep this maintained and up to date with the original version.

Reply:
I created the example for this dp relaying on the resources created and Composer Env Variable set. Do you suggest a better approach for this example?